src/kata1.fizzBuzz.js

```diff
@@ -1,5 +1,14 @@
 const fizzBuzz = (number) => {
-
-}
+    if (number % 5 === 0 && number % 3 === 0) {
+        return 'FizzBuzz'
+    } 
+    if (number % 5 === 0) {
+            return 'Buzz'
+        } if (number % 3 === 0) {
```

Comment:
Bring this `if` statement down onto a new line. Only `else if` and `else` should trail.